bc/channel/management/commands/mastodon-unsubscribe.py

```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
```

Comment:
I don't think We'll use this command that often. I used it once to delete the mastodon subscription and test the exception that is raised if the environment is missing the private and public keys but I haven't used it again since then.